src/core/contracts/registry.test.ts

```diff
@@ -51,7 +51,7 @@ describe("registry", () => {
     });
 
     it("Returns the correct id", async () => {
-      expect(givenId).toEqual("dsnp://0x00000000000003e8");
+      expect(givenId).toEqual("0x03e8");
```

Comment:
Wait, I thought the plan was to only ever return easier-to-use URIs for DSNPIds to the end user, not unlabelled hex strings. This string is ultimately returned to the user.

Am I remembering incorrectly or was there some change of plans?

Reply:
fixed